website/pages/blog/product-updates-1.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "CloudQuery Product Updates #1"
+tag: product
+date: 2022/11/01
+description: >-
+  Monthly updates on CloudQuery product and roadmap.
+author: yevgenypats
+---
+
+import { BlogHeader } from "../../components/BlogHeader"
+
+<BlogHeader/>
+
+Hey everyone! Since CloudQuery [V1 is out](./cloudquery-v1-release.md) we will be restarting our monthly products updates and starting from #1 as I figured it is easier to use numbers than months :)
+
+This month we've been really cranking it at CQ so lots of good stuff!
+
+## CLI Improvements
+
+* We've introduced wildcard matching to specify tables. Check out the [release blog](https://www.cloudquery.io/blog/introducing-wildcard-matching-for-tables)
+
+## Destinations
+
+We've added two new destinations to CQ!
+
+* [CSV](https://github.com/cloudquery/cloudquery/blob/main/plugins/destination/csv/README.md). See [Release Blog](https://www.cloudquery.io/blog/scaling-out-elt-with-cq-and-csv)
```

Comment:
Is it `release blog` (as in the previous section) or `Release Blog` as in here (and below?)